modules/installation-dns-user-infra.adoc

```diff
@@ -27,26 +32,127 @@ name and `<base_domain>` is the cluster base domain that you specify in the
 
 .2+a|Kubernetes API
 |`api.<cluster_name>.<base_domain>.`
-|This DNS A/AAAA or CNAME record must point to the load balancer
-for the control plane machines. This record must be resolvable by both clients
+|Add a DNS A/AAAA or CNAME record, and a DNS PTR record, to identify the load balancer
```

Comment:
Just a nitpicky suggestion, take it or leave it for all of the following additions: s/Add/Adds

Reply:
I went with add to sound a little more imperative.